apis/binding/v1alpha1/servicebinding_types.go

```diff
@@ -116,58 +124,58 @@ type Ref struct {
 type NamespacedRef struct {
 	Ref `json:",inline"`
 
-	// Namespace of the referent.
-	// if empty assumes the same namespace as ServiceBinding
+	// Namespace of the referent.  If unspecified, assumes the same namespace as
+	// ServiceBinding.
 	// +optional
 	Namespace *string `json:"namespace,omitempty"`
 }
 
-// Service defines the selector based on resource name, version, and resource kind
+// Service defines the selector based on resource name, version, and resource kind.
 type Service struct {
 	NamespacedRef `json:",inline"`
 
 	Id *string `json:"id,omitempty"`
 }
 
-// Application defines the selector based on labels and GVR
+// Application defines the selector based on labels and group version resource.
 type Application struct {
 	Ref `json:",inline"`
 	// +optional
 	LabelSelector *metav1.LabelSelector `json:"labelSelector,omitempty"`
 
-	// BindingPath refers to the paths in the application workload's schema
-	// where the binding workload would be referenced.
-	// If BindingPath is not specified the default path locations is going to
-	// be used.  The default location for ContainersPath is
-	// going to be: "spec.template.spec.containers" and if SecretPath
-	// is not specified, the name of the secret object is not going
-	// to be specified.
+	// Refers to the paths in the application workload's schema where the
+	// binding workload would be referenced.  If BindingPath is not specified,
+	// then the default path locations are used.  The default location for
+	// ContainersPath is "spec.template.spec.containers".  If SecretPath is not
+	// specified, then the name of the secret object does not need to be
+	// specified.
 	// +optional
 	BindingPath *BindingPath `json:"bindingPath,omitempty"`
 }
 
 // BindingPath defines the path to the field where the binding would be
-// embedded in the workload
+// embedded in the workload.
 type BindingPath struct {
-	// ContainersPath defines the path to the corev1.Containers reference
+	// ContainersPath defines the path to the corev1.Containers reference.
 	// If BindingPath is not specified, the default location is
-	// going to be: "spec.template.spec.containers"
+	// "spec.template.spec.containers".
 	// +optional
 	ContainersPath string `json:"containersPath"`
 
-	// SecretPath defines the path to a string field where
-	// the name of the secret object is going to be assigned.
-	// Note: The name of the secret object is same as that of the name of SBR CR (metadata.name)
+	// SecretPath defines the path to a string field where the name of the
+	// secret object is going to be assigned.  Note: The name of the secret
+	// object is same as that of the name of service binding custom resource
+	// (metadata.name).
 	// +optional
 	SecretPath string `json:"secretPath"`
 }
 
-// ServiceBinding expresses intent to bind an operator-backed service with
-// an application workload.
+// ServiceBinding expresses intent to bind an operator-backed service with an
```

Comment:
I would drop `operator-backed` from the sentence because we can bind to any service.

Reply:
Done in 71484a9.